### PR TITLE
openshift-metal3/dev-scripts: Add base Assisted Installer ZTP job

### PR DESCRIPTION
**Pull request by lranjbar**
Adding base ZTP job for Assisted Installer / infrastructure-operator. This job uses parts of the dev-scripts flow so this is to catch
potential breakages.

### CODE REVIEW

File: ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master__ai.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      version: "4.8"
+  initial:
+    integration:
+      name: "4.9"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.9"
+      namespace: ocp
```
Comment (andfasano) [inline ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master__ai.yaml:72]: Shouldn't be "4.11" the above releases?

**Comment by lranjbar** [comment]
This will be obsolete with the approach described in AGENT-170.

**Pull request closed without merging**